modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -176,6 +176,83 @@ mvn deploy
 
+== Publish an Asset with Mutable Data
+
+This example shows how to create your asset and set the mutable data describing it in the same request. The mutable data of an asset includes tags, custom fields, categories, and documentation pages.
+
+Declare tags, custom fields, and categories in the `pom.xml` file under the properties section, with the XML property `key` set to the key of the custom field or category:
+
+[source,xml,linenums]
+----
+    ...
+    <properties>
+        <categories key="categoryKey">someValue</categories>
+        <fields key="fieldKey">someValue</fields>
+        <tags>tag1,tag2,tag3</tags>
+    </properties>
+    ...
+----
+
+To set documentation pages, create a documentation directory under `src/main/resources`. The documentation directory contains each page as a Markdown file such as `Getting Started.md`. Replace `Getting Started` with the title of the page. Include resources such as images in the optional subdirectory `resources`. Specify the page order with the property `pageOrder` on the root of the main object in the optional file `config.json`:
+
+[source,console]
+----
+{
+ "pageOrder": ["home", "Getting Started", "Examples"]
+}
+----
+
+Add the Maven Assembly Plugin in the `pom.xml` plugin section to package the documentation folder, and create a Maven Assembly Plugin configuration file such as this:
+
+[source,xml,linenums]
+----
+<plugin>
+    <groupId>org.apache.maven.plugins</groupId>
+    <artifactId>maven-assembly-plugin</artifactId>
+    <version>2.6</version>
+    <configuration>
+        <appendAssemblyId>true</appendAssemblyId>
+    </configuration>
+    <executions>
+        <execution>
+            <id>create-distribution</id>
+            <phase>package</phase>
+            <goals>
+                <goal>single</goal>
+            </goals>
+            <configuration>
+                <descriptors>
+                    <descriptor>/src/main/resources/docs-assembly/assembly.xml</descriptor>
+                </descriptors>
+            </configuration>
+        </execution>
+    </executions>
+</plugin>
+----
+
+Create an `assembly.xml` file under `src/main/resources/docs-assembly` such as this:
+
+[source,xml,linenums]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.3"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.3 http://maven.apache.org/xsd/assembly-1.1.3.xsd">
+   <id>docs</id>
+   <formats>
+       <format>zip</format>
+   </formats>
+   <includeBaseDirectory>false</includeBaseDirectory>
+   <fileSets>
+       <fileSet>
+           <directory>${project.basedir}/src/main/resources/docs</directory>
+           <outputDirectory>/</outputDirectory>
+           <useDefaultExcludes>true</useDefaultExcludes>
+       </fileSet>
+   </fileSets>
+</assembly>
+----
+
```

Comment:
Could we add a link to: https://github.com/mulesoft-labs/exchange-documentation-samples/tree/master/mule-app-with-docs-and-tags ?
It could be a good example

Reply:
Great idea! Added.